modules/solid_properties/include/userobjects/ThermalUCProperties.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "ThermalSolidProperties.h"
+
+class ThermalUCProperties : public ThermalSolidProperties
+{
+public:
+  static InputParameters validParams();
+
+  ThermalUCProperties(const InputParameters & parameters);
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Woverloaded-virtual"
```

Comment:
Why is this required?

Reply:
Short answer: it is not. If the overload is intentional a `using Base::member` declaration should be added.